wiki/models/article.py

```diff
@@ -290,7 +297,7 @@ def save(self, *args, **kwargs):
             not self.previous_revision and
             self.article and
             self.article.current_revision and
-            self.article.current_revision != self):
+                self.article.current_revision != self):
```

Comment:
Too much indentation?

Reply:
What's the standard for this type of thing? I'll just do one line.

Reply:
Scratch that. Gonna just put it back 🤕